ios/Video/RCTVideo.swift

```diff
@@ -1195,7 +1195,8 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
 
     // MARK: - React View Management
 
-    func insertReactSubview(view: UIView!, atIndex: Int) {
+    override func insertReactSubview(_ view: UIView!, at atIndex: Int) {
+        super.insertReactSubview(subview, at:atIndex)
```

Comment:
What is `subview` pointing to? Should `view` in the function definition be renamed to `subview`?

Reply:
Thanks! Yeah, forgot to move this over from my local changes.